src/api/todos.js

```diff
@@ -0,0 +1,8 @@
+const todos = () => {
```

Comment:
Сделай одну функцию, которая будет принимать адрес

Reply:
```suggestion
const getTodos = () => {
```
название todos ни о чем не говорит